### DISCUSSION TITLE
deepmd-kit 2.0.2, error of init-frz-model command

### DISCUSSION THREAD

**Discussion by zjd11** [deepmodeling/deepmd-kit#1185] [Q&A]
I use init-frz-model command to train a frozen model generated by this command, and i find the newly trained model is the same as the old one, because i use the two models to perform a MD calculation, the result is same. I have tested different "auto_prob_style":  "prob_sys_size; when i train the new model. How can i fix the error.

**Comment by njzjz** [comment]
Did you freeze the model again?

**Reply by zjd11** [reply]
yes, when i finished the train, i freeze the model again.

**Reply by njzjz** [reply]
Sorry, I cannot reproduce your issue. Could you provide an example for reproducing?

**Reply by zjd11** [reply]
I have uploaded an example, including data set, input.json, and so on. I find, in deepmd-kit-2.0.2, i use  'dp train -ini-freeze-model' command to train a model based on a compressed one, the obtained model is the same as the original one，because the lammps output is the same.  but when i use the command to train a model based on a uncompressed one, the results are different.
[202.zip](https://github.com/deepmodeling/deepmd-kit/files/7286890/202.zip)

**Reply by njzjz** [reply]
I can reproduce the phenomenon. cc @denghuilu

**Reply by damonturney** [reply]
Was this error fixed?

**Comment by zjd11** [comment]
yes, when i finished the train, i freeze the model again.
